systemd/pivot.service.in

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Pivot Tool
+ConditionPathExists=/etc/pivot/image-pullspec
+After=ignition-firstboot-complete.service
+
+[Service]
+Type=simple
+ExecStart=@@PIVOT_BINARY_PATH@@
+Restart=on-failure
+RestartSec=30
```

Comment:
Though this is something we should discuss. In the MCD path, we don't want systemd to retry forever and we already have our own retry wrapper. In the (for now unnecessary) pivot on boot path, we do want this. Or maybe we should just drop this here and the wrapper in the MCD and actually add retry logic within pivot itself for things that can fail transiently (notably, all the operations that involving fetching data remotely)?